src/main/java/org/opentripplanner/middleware/utils/ConfigUtils.java

```diff
@@ -25,14 +25,14 @@
     private static final Logger LOG = LoggerFactory.getLogger(ConfigUtils.class);
 
     public static final String DEFAULT_ENV = "configurations/default/env.yml";
-    public static final String DEFAULT_ENV_SCHEMA = "src/main/resources/env.schema.json";
+    public static final String DEFAULT_ENV_SCHEMA = "env.schema.json";
     private static JsonNode ENV_SCHEMA = null;
     private static final String JAR_PREFIX = "otp-middleware-";
 
     static {
         // Load in env.yml schema file statically so that it is available for populating properties when running CI.
         try {
-            ENV_SCHEMA = yamlMapper.readTree(new FileInputStream(DEFAULT_ENV_SCHEMA));
+            ENV_SCHEMA = yamlMapper.readTree(ConfigUtils.class.getClassLoader().getResourceAsStream(DEFAULT_ENV_SCHEMA));
```

Comment:
I think this load/read stuff should go into its own method (it's used here and in the Updater class) and the javadoc should state that it must be loaded as a resource. Otherwise, we lose any context as to why this change was made and run the risk of someone changing it back to a `new FileInputStream` because it looks nicer.

Reply:
Updated.